### ISSUE TITLE
Query regarding BinaryAI plugin for Ghidra

### ISSUE THREAD

**Issue by Dmitri1998** [binaryai/plugins#2]
Hi,
I was trying to setup `BinaryAI` plugin with Ghidra and got results. Are the functions tagged as `BinaryAI` the matching/similar functions to the functions in the binary? I see that the json contains the matching function names with their `scores`, `file_path`, `entry_url` etc. I assume that the matching functions are computed from the known list of projects. Would it be possible to find similar functions among the custom set of functions locally?

**Comment by michellehong** [comment]
Hi Dmitri, thank you for your feedback!

> Are the functions tagged as BinaryAI the matching/similar functions to the functions in the binary? 

Yes.

> I see that the json contains the matching function names with their scores, file_path, entry_url etc. I assume that the matching functions are computed from the known list of projects.

You are right. We conduct function similarity search in a large range of C/C++ open source projects and provide users with the most similar source code functions. 

> Would it be possible to find similar functions among the custom set of functions locally?

We have plans to enable users to customize and would appreciate it if you can share the reason that you need a custom set of functions to help us understand your needs. Please stay tuned for our updates.😄

**Comment by Dmitri1998** [comment]
Hi,
So I was trying to setup the environment for the `Order Matters: Semantic-Aware Neural Networksfor Binary Code Similarity Detection` paper and noticed that the `binaryai/sdk` repo is no longer maintained. I am trying with a custom set of functions as I am not sure whether the function similarity seach would cover our experiments, and for binary diffing experiments, in which case we have to compare between 2 binaries.

**Comment by michellehong** [comment]
Hi, we understand your usage scenario now!
BinaryAI's scope of function sets cannot be customized by users now, but we will bring new features soon. Please stay tuned. 
If BinaryAI does not cover the open source projects that you need, please contact binaryai@tencent.com with the project information. Thanks for your feedback!

**Comment by Dmitri1998** [comment]
Hi,
Thanks for the previous reply. I see that the jsons that I am downloading from [https://www.binaryai.net/](url) contain lesser number of functions than that in the binary. Can you confirm is this the case?

**Comment by michellehong** [comment]
> Hi, Thanks for the previous reply. I see that the jsons that I am downloading from [https://www.binaryai.net/](url) contain lesser number of functions than that in the binary. Can you confirm is this the case?

Yes, we only list functions that has a perfect match in the json.
